src/helpers/git_helpers/remote_helpers.sh

```diff
@@ -3,7 +3,12 @@
 
 # Returns the url for the remote
 function remote_url {
-  git remote -v | grep "origin.*fetch" | awk '{print $2}'
+  mock_remote=$(git config --get git-town.testing.remote-url)
```

Comment:
I also don't like this, but I see no other way to go about this. I have added a `GT_ENV` variable that must be set to `test` in order for the mock to be considered here.